static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -0,0 +1,151 @@
+.newsletters-heading {
+    @include fs-header(5);
+    padding: $gs-baseline 0;
+
+    @include mq($from: tablet) {
+        font-size: 36px;
+        margin-top: 8px;
+  }
+}
+
+.newsletter-card__signup {
+    padding: .5rem;
+}
+
+.newsletter-card__example {
+    display: inline;
+    float: right;
+
+    .newsletter-card__lozenge {
+        color: $news-main-1;
+        border-color: $news-main-1;
+        -webkit-border-radius: 62.5rem;
+        border-radius: 100px;
+        height: 2rem;
+        width: 6rem;
+        margin: 0;
+
+        &:hover,
+        &:focus,
+        &:active {
+            border-color: lighten($news-main-1, 10%);
+        }
+    }
+}
+
+.newsletter-card__wrapper {
+    position: relative;
+
+    @include mq(desktop) {
+        margin-left: 17%;
+    }
+}
+
+.newsletter-card {
+    color: $neutral-1;
+    display: inline-block;
+    float: left;
+    height: 16.5rem;
+    width: 90%;
+    margin: $gs-baseline 5%;
+    background-color: $neutral-6;
+    border-top: $gs-baseline/4 solid $news-main-2;
+    border-bottom: $gs-baseline/4 solid $neutral-3;
+
+    @include mq(tablet) {
+        height: 18rem;
+        width: 20rem;
+        margin: $gs-baseline;
+    }
+
+    @include mq(desktop) {
+        height: 21rem;
+        width: 30%;
```

Comment:
if possible I would try and reuse the facia classes for columns, e.g. `l-row  l-row--cols-3` on the parent and `l-row__item l-row__item--span-1` on children - it might save some CSS. have a look at styles on the fronts and `static/src/stylesheets/module/facia/_l-list.scss`